server/src/main/java/org/apache/druid/server/coordinator/ReplicationThrottler.java

```diff
@@ -36,74 +39,89 @@
 {
   private static final EmittingLogger log = new EmittingLogger(ReplicationThrottler.class);
 
-  private final Map<String, Boolean> replicatingLookup = new HashMap<>();
```

Comment:
Change summary:
Commoned out `ReplicationSegmentHolder` as `TierLoadingState` to be used for tracking both `currentlyReplicatingSegments` and `currentlyMovingSegments` (in `SegmentStateManager`).